hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -288,12 +337,12 @@ void whiteBoxEncodingErrors() {
                 oneByte,
                 BigInteger.ONE,
                 oneByte,
-                null,
+                oneByte,
                 1,
                 oneByte,
                 oneByte,
                 oneByte);
-        assertThrows(IllegalStateException.class, ethTsDataEIP2930::encodeTx);
+        assertThrows(IllegalStateException.class, ethTxDataWithAccessList::encodeTx);
```

Comment:
This should probably be `ethTsDataEIP2930`? We could also rename it to `ethTxDataEIP2930`